src/component.vue

```diff
@@ -386,9 +394,9 @@ export default {
 
       if (this.arrowsIndex === null) {
         this.selectedItem = this.itemsComputed[0]
-
-        this.fireSelectEvent(this.selectedItem)
       }
+
+      this.fireSelectEvent(this.selectedItem)
```

Comment:
It should be as it was. The reason is that when the user selects with arrows, the selection occurs and the "select" event is triggered. 

Imagine the situation that the user selected the desired element through the arrows, the "selet" event or even a few fired, then user presses Enter which does not selects an element, but simply closes the element (`this.focused = !this.focused`), in which case the extra "select" event will be called.

Reply:
You can do this:

```js
  if (this.arrowsIndex === null) {
    this.selectedItem = this.itemsComputed[0]
    this.fireSelectEvent(this.selectedItem)	
  } else {
    if (this.disableAutoSelect) {
      this.fireSelectEvent(this.selectedItem)
    }
  }


```